packages/synapse-react-client/src/components/StorybookComponentWrapper.tsx

```diff
@@ -115,17 +115,19 @@ export function StorybookComponentWrapper(props: {
   )
 
   return (
-    <QueryClientProvider client={storybookQueryClient}>
-      <SynapseContextProvider synapseContext={synapseContext}>
-        {storybookContext.globals.showReactQueryDevtools && (
-          <ReactQueryDevtools />
-        )}
-        <MemoryRouter>
-          <SynapseToastContainer />
-          <main>{props.children}</main>
-        </MemoryRouter>
-      </SynapseContextProvider>
-    </QueryClientProvider>
+    <Suspense fallback={'global suspense loading...'}>
```

Comment:
Add global suspense to stories so Storybook doesn't crash when rendering a component that is not wrapped in suspense

Reply:
Will a fallback be available in SWC and SRC-based portals?